toolkit/scss/forms/_summary.scss

```diff
@@ -179,7 +260,6 @@ a.summary-item-remove-link {
 
     span {
       display: block;
-      padding-left: 10px;
```

Comment:
Yeah, got too enthusiastic about removing padding from everything.  Adding it back in.